ops/framework.py

```diff
@@ -518,7 +518,11 @@ def __getattr__(self, name: str) -> BoundEvent:
 
 
 class LifecycleEvent(EventBase):
-    """Events tied to the lifecycle of the Framework object."""
+    """Events tied to the lifecycle of the Framework object.
+
+    Note: Here, "lifecycle" was poorly named and has nothing to do with the
```

Comment:
Do you think this should be a Sphinx `note::`? Or does that draw too much attention to it?

Reply:
indeed note:: is a bit too prominent.

I also tried inset text, but our doc linter baulked at that.